docs/en/ufs/Minio.md

```diff
@@ -33,8 +33,8 @@ Minio is an object storage server built for cloud applications and DevOps. Minio
 source alternative to AWS S3.
 
 Launch a Minio server instance using the steps mentioned
-[here](http://docs.minio.io/docs/minio-quickstart-guide). Then create a bucket (or use an existing
-bucket). Once the server is launched, keep a note of Minio server endpoint, accessKey and secretKey.
+[here](http://docs.minio.io/docs/minio-quickstart-guide). Then either create or use an existing
```

Comment:
`create an existing bucket` doesn't make sense?

Reply:
But the text reads "Then either create or use an existing bucket.."

Reply:
`either create a new bucket or use an existing bucket` would make sense to me.
The current wording implies `create` and `use` both act on `an existing bucket`

Reply:
I've updated the wording